streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -2661,17 +2662,20 @@ public void shouldNotCheckpointAfterRestorationWhenExactlyOnceEnabled() {
     }
 
     @Test
-    public void shouldPunctuateNotHandleFailProcessingExceptionAndThrowStreamsException() {
+    public void punctuateShouldNotHandleFailProcessingExceptionAndThrowStreamsException() {
```

Comment:
renaming to make it readable :) (same below)

Plus some formatting changes to make the code readable